lessons/ro/chapter_9.yaml

```
@@ -0,0 +1,177 @@
+- title: Capitolul 9 - Organizarea și Structura Proiectului
+  content_markdown: >
+    Până acum, toate exemplele noastre de cod au fost un singur fișier. Haideți să discutăm cum
+    codul nostru poate fi mai bine organizat și partajat de alții!
+- title: Module
+  content_markdown: |
+    Fiecare program sau bibliotecă Rust este o *cutie*.
+
+    Fiecare cutie este alcătuit dintr-o ierarhie de *module*.
+
+    Fiecare cutie are un modul rădăcină.
+
+    Un modul poate conține variabile globale, funcții, structuri, trăsături sau chiar alte module!
+
+    În Rust nu există o corespondență 1 la 1 între fișiere și ierarhia de module. Trebuie să construim arborele de module în mod explicit și manual în codul nostru.
+- title: Scrierea unui Program
+  content_markdown: |
+    Un program are un modul rădăcină într-un fișier numit `main.rs`.
+- title: Scrierea unei Biblioteci
+  content_markdown: |
+    O bibliotecă are un modul rădăcină într-un fișier numit `lib.rs`.
+- title: Referințierea la alte Module și Cutii
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use%20std%3A%3Af64%3A%3Aconsts%3A%3API%3B%0A%0Afn%20main()%20%7B%0A%20%20%20%20println!(%22Welcome%20to%20the%20playground!%22)%3B%0A%20%20%20%20println!(%22I%20would%20love%20a%20slice%20of%20%7B%7D!%22%2C%20PI)%3B%0A%7D%0A
+  content_markdown: >
+    Elementele din module pot fi refererențiate cu calea completă a acestora.
+    `std::f64::consts::PI`.
+
+
+    O modalitate mai simplă este cuvântul cheie **use**. Acesta ne permite să specificăm
+    anumite elemente din modulele pe care dorim să le
+
+    să folosim în tot codul nostru fără o cale completă. Spre exemplu `use
+    std::f64::consts::PI`
+
+    îmi permite să folosesc doar identificatorul `PI` în funcția mea principală.
+
+
+    **std** este o cutie a **bibliotecii standard** din Rust, care este plină de
+    date utile
```

```suggestion
    date utile,
```

What is *date utile*?

---

Mistranslation on my part due to spacing, should be fixed in latest commit.